csrc/engine/assignment.cpp

```diff
@@ -7,26 +7,13 @@
 #include "assignment.h"
 
 namespace slime {
-std::vector<Assignment> Assignment::split(int step)
+std::string Assignment::dump()
```

Comment:
Not to change in this PR, but we can implement `operator<<` so it is easier for `cout` directly.